lib/hooks/http/middleware/defaults.js

```diff
@@ -146,27 +146,11 @@ module.exports = function(sails, app) {
       return res.send(400, bodyParserFailureErrorMsg);
     },
 
-    // By default, the express router middleware is installed towards the end.
-    // This is so that all the built-in core Express/Connect middleware
-    // gets called before matching any explicit routes or implicit shadow routes.
-    router: app.router,
-
     // Add powered-by Sails header
     poweredBy: function xPoweredBy(req, res, next) {
```

Comment:
Can we turn this off? It's a potential security hole to announce the server software. Express4 (and maybe 3 too) allow it to be disabled with `app.set('x-powered-by', false)` (see [docs](http://expressjs.com/en/4x/api.html)).